CHANGELOG.md

```diff
@@ -1,8 +1,3 @@
-## 0.10.0 (TBD)
-
-* Added more `RpoDigest` and `RpxDigest` conversions (#311).
-* Migrated to Winterfell v0.9.0 (#315)
-
```

Comment:
We should probably rebase this PR from the latest `next` to make sure we carry over the latest updates.